lib/CMakeLists.txt

```diff
@@ -1,13 +1,12 @@
 
 set ( PC_SOURCES
-        hashtable.c 
-        stringbuffer.c      
-        pc_bytes.c       
-        pc_dimstats.c      
-        pc_filter.c    
+        hashtable.c
+        stringbuffer.c
+        pc_bytes.c
+        pc_dimstats.c
+        pc_filter.c
```

Comment:
Can you please remove the above changes? I am picky, but I want to avoid pulling changes that should be done upstream.